src/libs/OptionsListUtils.ts

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable no-continue */
 import {parsePhoneNumber} from 'awesome-phonenumber';
 import Str from 'expensify-common/lib/str';
+// eslint-disable-next-line you-dont-need-lodash-underscore/get
 import lodashGet from 'lodash/get';
```

Comment:
> @kubabutkiewicz I see only times implementation, but I'm not sure if the new folder is a right place for that. But overall I guess you can just get rid of lodashGet, lodashSet. And if you afraid of regressions you can leave usage of lodashOrderBy, lodashSortBy, and maybe even lodashTimes
cc @blazejkustra

I agree with @VickyStash, why lodashGet was left here?

Reply:
@blazejkustra It's because of [this line](https://github.com/Expensify/App/blob/52278e7a83eae64b7b800873f7e65d9e010f72d8/src/libs/OptionsListUtils.ts#L796), we are getting the value dynamically.